client/account-recovery/lost-password/lost-password-form/style.scss

```diff
@@ -0,0 +1,22 @@
+.lost-password-form__title {
+	font-size: 1.5em;
+	font-weight: bold;
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/8955/commits/8c33b88f62f87d79ed100ebfe60566134ff232d0